.github/ISSUE_TEMPLATE/1-lts-release-checklist.md

```diff
@@ -35,7 +38,7 @@ This role should rotate between LTS releases
 
 - [ ] Backporting announcement email - [generate-backporting-announcement script](https://github.com/jenkins-infra/backend-commit-history-parser/blob/master/bin/generate-backporting-announcement)
 
-- [ ] Update jira labels with the selected issues, e.g. `2.277.2-fixed`, `2.277.2-rejected`
+- [ ] Update jira labels with the selected issues, e.g. `2.277.2-fixed`, `2.277.2-rejected`, and remove `lts-candidate`
```

Comment:
Ambiguous: if `2.xxx.y-rejected` is added, then `lts-candidate` (typically) remains.